src/services/devHandler/triggerDailyScoring.js

```diff
@@ -0,0 +1,16 @@
+const {successResponse, errorResponse} = require('../../utils');
+const {onDailyProcessTrigger} = require('../../processes/scoring-daily-process');
+
+const triggerDailyScoring = async (req, res) => {
+  try {
+    const result = await onDailyProcessTrigger();
+    return successResponse(res, 'triggerDailyScoring sucesfully', result);
+  } catch (error) {
+    console.log(error);
+    return errorResponse(res, error.toString(), 500);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in this function could be improved. Currently, the error is logged to the console and then sent as a response. This could potentially expose sensitive information to the client. Instead, consider logging the error and sending a generic error message to the client.

```diff
const triggerDailyScoring = async (req, res) => {
  try {
    const result = await onDailyProcessTrigger();
    return successResponse(res, 'triggerDailyScoring sucesfully', result);
  } catch (error) {
-   console.log(error);
-   return errorResponse(res, error.toString(), 500);
+   console.error(`Error in triggerDailyScoring: ${error}`);
+   return errorResponse(res, 'An error occurred while triggering daily scoring', 500);
  }
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->